src/Commonmark/Extensions/WikiLink.hs

```diff
@@ -263,7 +263,15 @@ wikilinkSpec =
  TODO: extend on top of plainify from heist-extra
 -}
 plainify :: [B.Inline] -> Text
-plainify = W.query $ \case
+plainify = W.query plainify' . W.walk (mapMaybe removeInlineNotes)
```

Comment:
What's the difference compared to just doing it as part of `query`?

<img width="329" alt="image" src="https://github.com/srid/commonmark-wikilink/assets/3998/d3ae6df1-57df-4286-b7aa-f0b5e1bd27c8">

Reply:
If I understand correctly, `query` always traverses the Note childs, while `walk` can be used to remove the Note from the AST.